app/models/admin.rb

```diff
@@ -1,18 +1,48 @@
 class Admin < ActiveRecord::Base
   include ActiveModel::ForbiddenAttributesProtection
-  # Authlogic gem
-  acts_as_authentic do |config|
-    config.transition_from_restful_authentication = true
-    if (ArchiveConfig.BCRYPT  || "true")  == "true" then
-      config.crypto_provider = Authlogic::CryptoProviders::BCrypt
-      config.transition_from_crypto_providers = [Authlogic::CryptoProviders::Sha512, Authlogic::CryptoProviders::Sha1]
-    else
-      config.crypto_provider = Authlogic::CryptoProviders::Sha512
-      config.transition_from_crypto_providers = [Authlogic::CryptoProviders::Sha1]
+
+  devise :database_authenticatable,
+         :registerable,
```

Comment:
I don't know how we currently handle Admin accounts, but :registerable means Admins can theoretically sign up themselves and also delete their own account? We may want to skip that for Admins. [devise_invitable](https://github.com/scambra/devise_invitable) may be worth a look, it's a gem to invite users via email, so only an existing admin can create a new admin.

I think we also need :rememberable if we allow Admins to stay signed in/use cookies.